examples/07_physics/resources/config.yml

```diff
@@ -0,0 +1,9 @@
+dimensions: null
+fullscreen: false
+max_dimensions: null
+min_dimensions: null
+multisampling: 1
+title: "Pong example"
```

Comment:
You probably want to change this. Noticed this now that I have ran it on a computer.